posthog/queries/test/test_trends.py

```diff
@@ -4029,79 +4032,204 @@ def _create_active_user_events(self):
                 properties={"key": "val"},
             )
 
-            _create_person(team_id=self.team.pk, distinct_ids=["p1"], properties={"name": "p1"})
             _create_event(
                 team=self.team,
                 event="$pageview",
                 distinct_id="p1",
                 timestamp="2020-01-09T12:00:00Z",
                 properties={"key": "val"},
             )
+            _create_event(
+                team=self.team,
+                event="$pageview",
+                distinct_id="p2",
+                timestamp="2020-01-09T12:00:00Z",
+                properties={"key": "val"},
+            )
+
             _create_event(
                 team=self.team,
                 event="$pageview",
                 distinct_id="p1",
                 timestamp="2020-01-10T12:00:00Z",
                 properties={"key": "val"},
             )
+
             _create_event(
                 team=self.team,
                 event="$pageview",
                 distinct_id="p1",
                 timestamp="2020-01-11T12:00:00Z",
                 properties={"key": "val"},
             )
-
-            _create_person(team_id=self.team.pk, distinct_ids=["p2"], properties={"name": "p2"})
             _create_event(
                 team=self.team,
                 event="$pageview",
                 distinct_id="p2",
-                timestamp="2020-01-09T12:00:00Z",
+                timestamp="2020-01-11T12:00:00Z",
                 properties={"key": "val"},
             )
+
             _create_event(
                 team=self.team,
                 event="$pageview",
-                distinct_id="p2",
-                timestamp="2020-01-11T12:00:00Z",
+                distinct_id="p0",
+                timestamp="2020-01-12T12:00:00Z",
                 properties={"key": "val"},
             )
 
-        def test_active_user_math(self):
-            self._create_active_user_events()
+        def test_weekly_active_users_daily(self):
```

Comment:
I think we should be using `@snapshot_clickhouse_queries` for this and other new tests. Also consider `@test_with_materialized_columns` if test is touching these.

Most of these predate the helper but this helps figure out the generated sql. For example I was really curious on what the shape of event_query is.

Reply:
Good suggestion, added `@snapshot_clickhouse_queries`